src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitRepository.kt

```diff
@@ -13,8 +14,22 @@ import javax.persistence.LockModeType
 @Repository
 interface VisitRepository : JpaRepository<Visit, Long>, JpaSpecificationExecutor<Visit> {
 
+  @Query(
+    "SELECT v.application_reference FROM visit v " +
+      "WHERE v.visit_status = 'RESERVED' and v.modify_timestamp < NOW() - (make_interval(mins => :expiredPeriodMinutes))",
+    nativeQuery = true
+  )
+  fun findExpiredApplicationReferences(expiredPeriodMinutes: Int): List<String>
```

Comment:
Change related to above comment, replace expiredPeriodMinutes with LocalDateTime?

Reply:
comment above